R/summarise.R

```diff
@@ -1,5 +1,19 @@
-
+#'waltz() takes a grouped tibble and a list of formulas and returns a tibble with: as many columns as supplied formulas,
+#'one row per group. It does not prepend the grouping variables (see tango for that).
```

Comment:
you can reference other functions with markdown: 

```suggestion
#'one row per group. It does not prepend the grouping variables (see [tango()] for that).
```